internal/authentication/oidc.go

```diff
@@ -16,6 +16,8 @@ import (
 	"github.com/go-kit/kit/log"
 	"github.com/go-kit/kit/log/level"
 	"golang.org/x/oauth2"
+
+	"github.com/observatorium/observatorium/internal"
```

Comment:
Just thought, not a blocker - it's best to specify some package inside internal. Just internal package sounds like `utils` to me (: